docs/devguide/flashfirmware.rst

```diff
@@ -1,58 +1,83 @@
 .. _flashfirmware:
 
-=================
-Flashing Firmware
-=================
+========================
+Build and flash firmware
+========================
 
-Building firmware
------------------
-Use yotta to build.
+micro:bit V1
+============
 
-Use target bbc-microbit-classic-gcc-nosd::
+This applies to MicroPython for the micro:bit V1, the source of which can be 
+found here: `bbcmicrobit/micropython <https://github.com/bbcmicrobit/micropython>`_.
 
-  yt target bbc-microbit-classic-gcc-nosd
+Dependencies
```

Comment:
I think these instructions depend on the Yotta installer, which might disappear and even when it was maintained it was usually outdated, we should update these to list the tools the users will need to install.

Reply:
done